bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -111,6 +118,81 @@ async def autoreview_status(self, ctx: Context) -> None:
         else:
             await ctx.send("Autoreview is currently disabled.")
 
+    @tasks.loop(hours=72)
+    async def track_forgotten_nominations(self) -> None:
+        """Track active nominations who are more than 2 weeks old."""
+        old_nominations = await self._get_forgotten_nominations()
+        nomination_details = await self._filter_out_tracked_nominations(old_nominations)
+        for nomination, nomination_vote_message in nomination_details:
+            issue_created = await self._track_vote_in_github(nomination)
+            if issue_created:
+                await nomination_vote_message.add_reaction(FLAG_EMOJI)
+
+    async def _get_forgotten_nominations(self) -> List[Nomination]:
+        """Get active nominations that are more than 2 weeks old."""
+        now = arrow.utcnow()
+        nominations = [
+            nomination
+            for nomination in await self.api.get_nominations(active=True)
+            if (now - nomination.inserted_at).days >= OLD_NOMINATIONS_THRESHOLD_IN_DAYS
+        ]
+        return nominations
+
+    async def _filter_out_tracked_nominations(
+            self,
+            nominations: List[Nomination]
+    ) -> List[Tuple[Nomination, discord.Message]]:
+        """Filter the forgotten nominations that are still untracked in GitHub."""
```

Comment:
This isn't just a filter, since it returns more data that it was given. This should be described in the docstring, and maybe change the function name to make it clear it isn't just a filter

Reply:
This isn't resolved.